iroha_data_model/src/events.rs

```diff
@@ -267,8 +382,15 @@ pub mod pipeline {
     /// Exports common structs and enums from this module.
     pub mod prelude {
         pub use super::{
-            EntityType as PipelineEntityType, Event as PipelineEvent,
-            EventFilter as PipelineEventFilter, Status as PipelineStatus,
+            BlockRejectionReason as PipelineBlockRejectionReason, EntityType as PipelineEntityType,
+            Event as PipelineEvent, EventFilter as PipelineEventFilter,
+            InstructionExecutionFail as PipelineInstructionExecutionFail,
+            NotPermittedFail as PipelineNotPermittedFail,
+            RejectionReason as PipelineRejectionReason,
+            SignatureVerificationFail as PipelineSignatureVerificationFail,
+            Status as PipelineStatus,
+            TransactionRejectionReason as PipelineTransactionRejectionReason,
+            UnsatisfiedSignatureConditionFail as PipelineUnsatisfiedSignatureConditionFail,
```

Comment:
It is not necessary to add `Pipeline` prefix to rejection reasons. I would simply use `TransactionRejectionReason` and `BlockRejectionReason`